### PR TITLE
docs: add information about the dynamic messages

### PR DESCRIPTION
**Pull request by kelsos**

### CODE REVIEW

File: messages/README.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+### Welcome message schema
+
+- `header` is an **optional** field. If it exists it will show a header to the message box.
+- `text` is a **required** field. It affects the body of text shown in the message box.
+- `action` is an **optional** field. If it exists it is required to have a `text` and a `url` property. Action will appear as a button, with text being the button text and url the link that will open when clicking it.
+- `icon` is an **optional** field. It has to be a link to an `svg` file, that is hosted in this repo.
+- `period` is a **required** field. It is the period for which the message will be displayed in the application welcome area.
+
+> If multiple messages overlap during a specific period only the first message will be shown.
+
```
Comment (yabirgb) [inline messages/README.md:46]: Why multiple messages? To let them as in a "queue" so they can automatically appear when the period ends?

**Review by yabirgb — COMMENTED** [review]
small question

File: messages/README.md (hunk 1 above)
Comment (kelsos) [reply messages/README.md:46]: Yes so if we want to say schedule 3 different messages to show in different periods in month we don't need to remove one and add another. We can "schedule" them by adding them in the list.

**Review by yabirgb — APPROVED** [review]
lgtm

**Pull request merged**